### DISCUSSION TITLE
USB POWER TRIGGER not working on Hero 11

### DISCUSSION THREAD

**Discussion by ryantrax** [gopro/labs#521] [General]
I'm trying to get USB POWER TRIGGER to work on the Hero 11 Black. It works fine on my Hero 9 and 10, I just can't get it to work on the 11. I scan the QR it says it's accepted and powers down, I add USB power and it doesn't power on and start recording. I'm using command !uN!S!u10E!R just like on my Hero 9 and 10. 
Is this a known issue with the last update?

**Comment by dnewman-gpsw** [comment]
As this is a confirmed bug, I added it to issue here: https://github.com/gopro/labs/issues/523

I also list a workaround for you to try.

**Reply by ryantrax** [reply]
Thank you, the workaround worked perfectly, I appreciate it!
